src/Components/CurrentBenefits/CurrentBenefits.tsx

```diff
@@ -0,0 +1,226 @@
+import { ReactComponent as FamilyPlanning } from '../../Assets/OptionCardIcons/AcuteConditions/family_planning.svg';
+import { ReactComponent as JobResources } from '../../Assets/OptionCardIcons/AcuteConditions/job_resources.svg';
+import { ReactComponent as DentalCare } from '../../Assets/OptionCardIcons/AcuteConditions/dental_care.svg';
+import { ReactComponent as LegalServices } from '../../Assets/OptionCardIcons/AcuteConditions/legal_services.svg';
+import './CurrentBenefits.css';
+
+type CategoryName =
+  | 'Housing and Utilities'
+  | 'Food and Nutrition'
+  | 'Health Care'
+  | 'Transportation'
+  | 'Tax Credits'
+  | 'Cash Assistance'
+  | 'Child Care, Youth, and Education'
+  | 'Food or Groceries'
+  | 'Diapers'
+  | 'Housing'
+  | 'Managing housing costs'
+  | 'Behavioral health'
+  | "Child's development"
+  | 'Family planning'
+  | 'Job training'
+  | 'Low-cost dental care'
+  | 'Civil legal needs';
+
+const isCategoryName = (maybeCategoryName: string | undefined): maybeCategoryName is CategoryName => {
+  if (!maybeCategoryName) {
+    return false;
+  }
+
+  return [
+    'Housing and Utilities',
+    'Food and Nutrition',
+    'Health Care',
+    'Transportation',
+    'Tax Credits',
+    'Cash Assistance',
+    'Child Care, Youth, and Education',
+    'Food or Groceries',
+    'Diapers',
+    'Housing',
+    'Managing housing costs',
+    'Behavioral health',
+    "Child's development",
+    'Family planning',
+    'Job training',
+    'Low-cost dental care',
+    'Civil legal needs',
+  ].includes(maybeCategoryName);
+};
+
+export const iconCategoryMap: Record<CategoryName, React.ComponentType> = {
+  'Housing and Utilities': Housing,
+  'Food and Nutrition': Food,
+  'Health Care': HealthCare,
+  Transportation: Transportation,
+  'Tax Credits': TaxCredits,
+  'Cash Assistance': CashAssistance,
+  'Child Care, Youth, and Education': ChildCareYouthEducation,
+  'Food or Groceries': FoodOrGroceries,
+  Diapers: BabySupplies,
+  Housing: Housing,
+  'Managing housing costs': ManagingHousingCosts,
+  'Behavioral health': BehavioralHealth,
+  "Child's development": ChildDevelopment,
+  'Family planning': FamilyPlanning,
+  'Job training': JobResources,
+  'Low-cost dental care': DentalCare,
+  'Civil legal needs': LegalServices,
+};
+
+type Program = {
+  name: Translation;
+  website_description: Translation;
+  category?: Translation;
+  id: number;
+  type?: Translation;
+};
+
+type Category = {
+  name: Translation;
+  programs: Program[];
+};
+
+const CurrentCOBenefits = () => {
+  const [allLongTermPrograms, setAllLongTermPrograms] = useState<Program[]>([]);
+  const [allNearTermPrograms, setAllNearTermPrograms] = useState<Program[]>([]);
+  const intl = useIntl();
+
+  useEffect(() => {
+    getAllLongTermPrograms().then((response) => {
+      setAllLongTermPrograms(response);
+    });
+
+    getAllNearTermPrograms().then((response) => {
+      setAllNearTermPrograms(response);
```

Comment:
We can do this to prevent a lot of complexity in determining whether to use type or category in a lot of the functions. Additionally, there will only be a one line change when we change the API.
```suggestion
      response.map((program) => program.category = program.type)
      setAllNearTermPrograms(response);
```